example/Cpp/src/Patterns/FullyConnected.lf

```diff
@@ -0,0 +1,31 @@
+// In this pattern, each node can send broadcast messages to all other nodes
```

Comment:
Suggest looking at the examples in `example/C/src/Patterns` for naming convention and documentation style. The naming convention is intended to lead the reader through the examples in a particular order.